contracts/ts/genMaciState.ts

```diff
@@ -142,7 +142,7 @@ export const genMaciStateFromContract = async (
   // Check whether each pollId exists
   assert(foundPollIds.includes(Number(pollId)), "Error: the specified pollId does not exist on-chain");
 
-  const pollContractAddress = pollContractAddresses[pollId];
+  const pollContractAddress = pollContractAddresses[Number(pollId)];
```

Comment:
Let's do the same for `pollContractsAddresses` here, so it will be Map with bigint as keys and string as values.